src/orga/subteams.rst

```diff
@@ -45,6 +45,41 @@ Members
 - Anthony Scopatz <scopatz@gmail.com>
 
 
+Finance Sub-Team
+=============================
+
+Role
+----
+The purpose of the finance sub-team is to provide a point of contact
+for financial and budgetary issues. This includes keeping core aware
+of the current conda-forge balance via ``core`` meetings and facilitating
+dispersal of funds.
+
+Charter
+-------
+Static
+
+Responsibility
+--------------
+The core responsibilities of this team are to
+
+- keep core aware of budgetary and financial matters pertaining to conda-forge
+- facilitate dispersal of funds
+- give core updates via the standing budget item at core meetings
+- work closely with the NumFOCUS point of contact to ensure smooth financial operations
+
+This team will operate via the following rules
+
+- only members of ``core`` can be on this sub-team
+- this sub-team will report its activity to core at all ``core`` meetings
```

Comment:
```suggestion
- this sub-team will report its activity to core at all attended ``core`` meetings
```
or something. Meaning, it's fine if (currently the both of) you are on vacation at the same time and no one reports ;).

Reply:
I was kind of hoping that we'd have enough members to cover the meetings, but I guess not.

Reply:
(To be fair, the listed members are probably the most consistently attending -- I really can't remember a meeting in which not at least one of you was there!)